src/cluster.c

```diff
@@ -212,6 +212,9 @@ int clusterLoadConfig(char *filename) {
             n = createClusterNode(argv[0],0);
             clusterAddNode(n);
         }
+        /* Format for the node address information: 
+         * ip:port[@cport][,hostname] */
```

Comment:
After thinking about this more, I don't like it. I'll put the hostname in a separate field. It's probably good to have named values here.